src/components/SurveyComponents/ManagementActivities.tsx

```diff
@@ -0,0 +1,96 @@
+import { FormSelect, Navigation, SurveyPageProps, SurveyStep } from '@components'
+import { scrollToRef } from '@utils'
+import { useEffect, useRef } from 'react'
+const percentageBins = ['0-20%', '20-40%', '40-60%', '60-80%']
```

Comment:
this has already been defined in `EngineeringActivities.tsx`, and thus as we are using it in more than one component, it is better to extract it to a constant in the constants folder and reuse it instead of redefining it in every component